util/autovector.h

```diff
@@ -356,42 +363,49 @@ class autovector {
   }
 
  private:
-  size_type num_stack_items_ = 0;  // current number of items
-  alignas(alignof(
-      value_type)) char buf_[kSize *
-                             sizeof(value_type)];  // the first `kSize` items
-  pointer values_;
+  static void destory(value_type* p, size_t n) {
```

Comment:
Should probably be renamed to `destroy` (typo in previous method name).
```suggestion
  static void destroy(value_type* p, size_t n) {
```

Reply:
Done, thanks for your correction!